src/os/KeyValueStore.cc

```diff
@@ -1931,13 +1935,13 @@ int KeyValueStore::_generic_write(StripObjectMap::StripObjectHeaderRef header,
         value.append_zero(header->strip_size-value.length());
 
       header->bits[iter->no] = 1;
+      header->updated = true;
```

Comment:
That handles only when bitmap is populated beyond the size of the object, this case handles the stripes which are being populated now in that range.